src/main/webapp/feed.html

```diff
@@ -9,6 +9,16 @@
     </head>
 
     <body onload="buildUI()">
+    <nav>
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/aboutus.html">About Our Team</a></li>
+        <li><a href="/community.html">Community</a></li>
+        <li><a href="/feed.html">Feed</a></li>
+        <li><a href="/stats.html">Stats</a></li>
+        <li><a href="/user-page.html">User Page</a></li>
+      </ul>
+    </nav>
```

Comment:
I am not sure why this is not a merge conflict but the current code inside feed.html in the master branch is a newer version than this.

Reply:
same thing with every other file in this commit. Did you git pull before uploading your changes?

Reply:
Yes. 

C:\Users\naled\codeu-starter-project\team-03-codeu>git pull origin images1dev
From https://github.com/emmanguyen28/team-03-codeu
 * branch            images1dev -> FETCH_HEAD
Already up to date.